tests/test_acceptance.py

```diff
@@ -81,6 +95,209 @@ def test_koswat_package_can_be_imported(self):
         except ImportError as exc_err:
             pytest.fail(f"It was not possible to import required packages {exc_err}")
 
+    @pytest.fixture(
+        name="t_10_3_surroundings_wrapper_fixture",
+        params=[(False, False), (False, True), (True, False), (True, True)],
+        ids=[
+            "Without ANY surrounding",
+            "With Infrastructure",
+            "With Obstacles",
+            "With Infrastructure and Obstacles",
+        ],
+    )
+    def _get_surroundings_wrapper_fixture(
+        self,
+        request: pytest.FixtureRequest,
+    ) -> Iterable[tuple[SurroundingsWrapper, Path]]:
+        _traject = "10_3"
+        # Shp locations file
+        _shp_file = test_data.joinpath(
+            "shp_reader",
+            "Dijkvak",
+            "Dijkringlijnen_KOSWAT_Totaal_2017_10_3_Dijkvak.shp",
+        )
+        assert _shp_file.is_file()
+
+        # Surroundings directory
+        _surroundings_analysis_path = test_data.joinpath(
+            "acceptance", "surroundings_analysis", _traject
+        )
+        assert _surroundings_analysis_path.is_dir()
+
+        # Create a dummy dir to avoid importing unnecessary data.
+        _dir_name = get_testcase_results_dir(request)
+        _temp_dir = test_results.joinpath(_dir_name, _traject)
+        if _temp_dir.exists():
+            shutil.rmtree(_temp_dir)
+        shutil.copytree(_surroundings_analysis_path, _temp_dir)
+
+        # Generate surroundings section File Object Model.
+        _include_obstacles, _include_infras = request.param
+        _surroundings_settings = SurroundingsSectionFom(
+            surroundings_database_dir=_temp_dir.parent,
+            constructieafstand=50,
+            constructieovergang=10,
+            buitendijks=False,
+            bebouwing=_include_obstacles,
+            spoorwegen=False,
+            water=False,
+        )
+
+        # Generate Infrastructures section file model
+        _infrastructure_settings = InfrastructureSectionFom(
+            infrastructuur=_include_infras,
+            opslagfactor_wegen=SurtaxFactorEnum.NORMAAL,
+            infrakosten_0dh=InfraCostsEnum.GEEN,
+            buffer_buitendijks=0,
+            wegen_klasse2_breedte=2,
+            wegen_klasse7_breedte=5,
+            wegen_klasse24_breedte=8,
+            wegen_klasse47_breedte=12,
+            wegen_onbekend_breedte=8,
+        )
+
+        # Generate wrapper
+        _importer = SurroundingsWrapperCollectionImporter(
+            infrastructure_section_fom=_infrastructure_settings,
+            surroundings_section_fom=_surroundings_settings,
+            selected_locations=[],
+            traject_loc_shp_file=_shp_file,
+        )
+        _surroundings_wrapper_list = _importer.build()
+        assert any(_surroundings_wrapper_list)
+
+        # Yield result
+        yield _surroundings_wrapper_list[0], _temp_dir
+
+        # Remove temp dir
+        # shutil.rmtree(_temp_dir)
```

Comment:
Restore this line?

Reply:
No, we want to keep the results so that the PO can compare them, I'll remove the irrelevant ones though (without infrastructures)